inc/ajax.class.php

```diff
@@ -433,14 +436,16 @@ static function createTabs(
                      }
                   });
                }
-
-               var tabs = ui.tab.parent().children();
-               if (tabs.length > 1) {
-                  var newIndex = tabs.index(ui.tab);
-                  $.get(
-                     '".$CFG_GLPI['root_doc']."/ajax/updatecurrenttab.php',
-                     { itemtype: '".addslashes($type)."', id: '$ID', tab: newIndex }
-                  );
+               // We need to manually set the current tab if the main event was prevented
+               if (event_prevented) {
```

Comment:
Should work without having to use a new var.
```suggestion
               if (event.defaultPrevented) {
```

Reply:
Done but with `event.isDefaultPrevented()` as it is a jquery event.